src/utils/common.ts

```diff
@@ -90,6 +93,18 @@ export const buildUrlOptions = (options: { [key: string]: string }): string => {
   return optionStr;
 };
 
+export const getOfferItemIndexByBestOffer = (gasUsd: (number | undefined)[], receiveAmount: number[]) => {
```

Comment:
not sure why we return index here and not just best offer?

Reply:
also make sure u define in comments how best offer is selected

Reply:
Yeah wrote a comment explaining that